driver-sync/src/test/functional/com/mongodb/client/ChangeStreamsTest.java

```diff
@@ -0,0 +1,280 @@
+            List<CommandEvent> events = commandListener.getEvents();
+
+            for (int i = 0; i < expectedEvents.size(); i++) {
+                CommandEvent expectedEvent = expectedEvents.get(i);
+                CommandEvent event = events.get(i);
+                if (expectedEvent instanceof CommandStartedEvent && event instanceof CommandStartedEvent) {
+                    BsonDocument eventCommand = ((CommandStartedEvent) event).getCommand();
+                    eventCommand.remove("$clusterTime");
+                    eventCommand.remove("lsid");
+
+                    BsonDocument eventChangeStream = eventCommand.getArray("pipeline",
+                            new BsonArray(singletonList(new BsonDocument())))
+                            .get(0).asDocument().getDocument("$changeStream", new BsonDocument());
+
+                    BsonDocument expectedChangeStream = ((CommandStartedEvent) expectedEvent).getCommand().getArray("pipeline",
+                            new BsonArray(singletonList(new BsonDocument())))
+                            .get(0)
+                            .asDocument().getDocument("$changeStream", new BsonDocument());
+
+                    if (expectedChangeStream.containsKey("startAtOperationTime") && eventChangeStream.containsKey("startAtOperationTime")) {
+                        eventChangeStream.put("startAtOperationTime", expectedChangeStream.get("startAtOperationTime"));
+                    } else {
+                        eventChangeStream.remove("startAtOperationTime");
+                    }
+                }
+
+                CommandMonitoringTestHelper.assertEventsEquality(singletonList(expectedEvent), singletonList(event));
+            }
+        }
+    }
+
+
+    private MongoCursor<ChangeStreamDocument<BsonDocument>> createChangeStreamCursor() {
+        String target = definition.getString("target").getValue();
+        List<BsonDocument> pipeline = new ArrayList<BsonDocument>();
+        for (BsonValue bsonValue : definition.getArray("changeStreamPipeline", new BsonArray())) {
+            pipeline.add(bsonValue.asDocument());
+        }
+
+        if (target.equals("client")) {
+            return mongoClient.watch(pipeline, BsonDocument.class).iterator();
+        } else if (target.equals("database")) {
+            return mongoClient.getDatabase(namespace.getDatabaseName()).watch(pipeline, BsonDocument.class).iterator();
+        } else if (target.equals("collection")) {
+            return mongoClient.getDatabase(namespace.getDatabaseName()).getCollection(namespace.getCollectionName())
+                    .watch(pipeline, BsonDocument.class).iterator();
+        } else {
+            throw new IllegalArgumentException(format("Unknown target: %s", target));
+        }
+    }
+
+    private void handleOperations() {
+        MongoClient localMongoClient = MongoClients.create(getMongoClientSettings());
```

Comment:
Can you just use `Fixture.getMongoClient()` here instead?

Reply:
Done.